### PR TITLE
Updated Pods Description in Readme.md File

### PR DESCRIPTION
**Pull request by vaibhavbisht06**
1. Cache
2. Floaty
3. Google-cast-SDK
4. Imaginary
5. Protobuf
6. Reachability Swift
7. SDWebImage
8. SwipeCellKit

### CODE REVIEW

**Comment by anubhavsingh16** [comment]
@vaibhavbisht06 I already send PR for update Pods and their description.

**Comment by vaibhavbisht06** [comment]
You have incorporated only 3 Pods but i have incorporated 8 pods. Give a look on PR description.

File: AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1030,15 +1033,15 @@
...
+				DEVELOPMENT_TEAM = BXD5262846;
 				ENABLE_BITCODE = NO;
 				INFOPLIST_FILE = AmahiAnywhere/Info.plist;
 				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = com.amahi.AmahiAnywhere;
+				PRODUCT_BUNDLE_IDENTIFIER = com.vaibhav.AmahiAnywhere;
```
Comment (cpg) [inline AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj:1044]: Please remove this change

File: AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1030,15 +1033,15 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_ENTITLEMENTS = AmahiAnywhere/AmahiAnywhere.entitlements;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = BXD5262846;
```
Comment (cpg) [inline AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj:1036]: Please remove this change

File: AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -1053,7 +1056,8 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_ENTITLEMENTS = AmahiAnywhere/AmahiAnywhere.entitlements;
-				CODE_SIGN_STYLE = Automatic;
+				CODE_SIGN_IDENTITY = "Apple Development";
```
Comment (cpg) [inline AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj:1059]: Please remove this change

File: AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -1053,7 +1056,8 @@
...
+				CODE_SIGN_STYLE = Manual;
```
Comment (cpg) [inline AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj:1060]: Please remove this change

File: AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -1062,8 +1066,9 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = com.amahi.AmahiAnywhere;
+				PRODUCT_BUNDLE_IDENTIFIER = com.vaibhav.AmahiAnywhere;
```
Comment (cpg) [inline AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj:1069]: Please remove this change

**Comment by cpg** [comment]
No answers or addressing the issues. Closing.

**Pull request closed without merging**